src/contexts/AuthContext.tsx

```diff
@@ -39,24 +44,44 @@ export const AuthProvider = ({ children }: PropsWithChildren) => {
       console.log('[Firebase] Authentication state changed:', user?.uid);
```

Comment:
I like this [Firebase] log indicator. Can we do this for all comments (firebase or other)?

Reply:
yea that's what i had in mind